protocol/avdl/chat1/common.avdl

```diff
@@ -15,7 +15,8 @@ protocol common {
     ATTACHMENT_2,
     EDIT_3,
     DELETE_4,
-    METADATA_5
+    METADATA_5,
+    TLFNAME_6
```

Comment:
But it's a regular message type so it can also appear later, making it a bit confusing. Can we just send a `MEATDATA_5` message as the first one, or even `NONE_0`?